src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,36 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int MAX_NUMBER_OF_TOTAL_COUNTS = 11;
+	private static final int STARTING_NUMBER_OF_TOTAL_COUNTS = -1;
```

Comment:
-1 w callCheck() to inna liczba niż return w peekaboo, muszą to być osobne stałe